addon/adapters/rest.js

```diff
@@ -303,9 +304,17 @@ const RESTAdapter = Adapter.extend(BuildURLMixin, {
 
   useFetch: computed(function() {
     let ENV = getOwner(this).resolveRegistration('config:environment');
-    let shouldUseFetch = (ENV && ENV._JQUERY_INTEGRATION) === false || jQ === undefined;
-
-    return shouldUseFetch;
+    // TODO: migrate to static build time detection ala https://github.com/embroider-build/embroider/issues/162
```

Comment:
Can you please file an issue

Reply:
https://github.com/emberjs/data/issues/6093